spec/support/fixtures.rb

```diff
@@ -37,18 +37,14 @@ def self.find_for_jwt_authentication(_sub)
 
   # A dummy revocation strategy which keeps the state in its instances
   class RevocationStrategy
-    attr_reader :revoked
+    @@revoked = []
```

Comment:
Why are we changing this file?

Reply:
I changed it because we were not testing when the strategy was a module/class as String.
```
def self.constantize_values(hash)
```
now we do.

So the code started failing with string in the mapping in dry-configuration 0.11